.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           GOPROXY: "https://proxy.golang.org"
         run: go mod download
       - name: Build
-        run: go build ./...
+        run: go build ./... && go install ./cmd/...
```

Comment:
why are you needing to go install here?

Reply:
These changes add a provider binary that needs to be on the `PATH` when running tests; this is the easiest way to achieve that.

Reply:
Somewhat related: why does this repo no longer use the `Makefile`? IMO it's bad practice for CI and local development workflows to diverge at their core.

Reply:
That's a pure error - I will fix that up with our CI scripts

Reply:
Cool. Thanks!